src/components/Note/Note.scss

```diff
@@ -58,7 +58,9 @@ $note__author-image-size: 26px;
 }
 
 .note:focus-within,
-.note:focus,
+.note:focus {
+  outline: 2px solid rgba(var(--accent-color-rgb), 0.5);
+}
 .note:hover {
   transform: scale(1.02);
   outline: none;
```

Comment:
is it intended behavior that the outline of a focused card is removed on hover? I think it looks weird

Reply:
I aggree, it looks better if it stays